crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1159,6 +1159,7 @@ impl fmt::Debug for World {
     }
 }
 
+#[allow(clippy::non_send_fields_in_send_ty)]
```

Comment:
We should really have a comment here that this is in fact unsound, and linking to #3519.

Reply:
added links to https://github.com/bevyengine/bevy/issues/3666